custom_components/powercalc/data/innr/BY 285 C/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly, 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.0:docker"
+    },
+    "name": "BY 285 C",
```

Comment:
This may be a more descriptive / marketed name. It's used for supported models listing. See 3rd column. https://github.com/bramstroker/homeassistant-powercalc/blob/master/docs/supported_models.md#Lights

Reply:
Ah, that's not what the addon is matching against for the model, is it just using the folder structure for that ?

Zigbee2Mqtt has `B22 bulb RGBW`  the description, would that do ?